thunder/executors/nvfuserex.py

```diff
@@ -30,7 +30,7 @@ def nvfuser_version() -> LooseVersion | None:
 
 
 def required_nvfuser_version() -> LooseVersion:
```

Comment:
If nvFuser is installed but the version is not new enough should Thunder issue a warning about this?

Reply:
I think an error would be also good.